### ISSUE TITLE
Transitions preserving state?

### ISSUE THREAD

**Issue by zshannon** [heestand-xyz/Noise#4]
When I change parameters like `speed` using an animated transition the random pattern abruptly resets. Is it possible to smoothly transition?

**Comment by heestand-xyz** [comment]
Yes, this is known problem of the package. I'll look into it and see if I can make it relative.

**Comment by heestand-xyz** [comment]
I've now fixed the issue. Speed is now relative.
